tools/project_manager/display_image.js

```diff
@@ -31,70 +35,106 @@ $(function () {
+            } else {
                 fixHead.append(theImageControl.controls);
-                stretchDiv.addClass("overflow-auto image-back").append(theImageControl.image);
-                theImageControl.setZoom();
-                break;
-            }
-            case "text": {
-                let theTextControl = textControl();
-                // round selector submits if changed
-                pageControlForm.append(roundSelect(true));
-                fixHead.append(theTextControl.controls);
-                theTextControl.textArea.prop("readonly", true);
-                stretchDiv.append(theTextControl.textArea);
-                break;
             }
-            case "imageText": {
-                let theImageControl = imageControl();
-                let theTextControl = textControl();
-                theTextControl.textArea.prop("readonly", true);
-                let imageDiv = $("<div>", {class: 'overflow-auto image-back'}).append(theImageControl.image);
-                let textDiv = $("<div>").append(theTextControl.textArea);
-                stretchDiv.append(imageDiv, textDiv);
-                let theSplitter = viewSplitter(stretchDiv);
-                pageControlForm.append(roundSelect(true));
-                fixHead.append(theImageControl.controls, theSplitter.buttons, theTextControl.controls);
-                theImageControl.setZoom();
-                // re-layout after drawing fixed div
-                theSplitter.mainSplit.reLayout();
-                break;
+            stretchDiv.addClass("overflow-auto image-back").append(theImageControl.image);
+            theImageControl.setZoom();
+            break;
+        }
+        case "text": {
+            let theTextControl = textControl();
+            // round selector submits if changed
+            pageControlForm.append(roundSelect(true));
+            fixHead.append(theTextControl.controls);
+            theTextControl.textArea.prop("readonly", !mentorMode);
+            stretchDiv.append(theTextControl.textArea);
+            break;
+        }
+        case "imageText": {
+            let theImageControl = imageControl();
+            let theTextControl = textControl();
+            theTextControl.textArea.prop("readonly", !mentorMode);
+            let imageDiv = $("<div>", {class: 'overflow-auto image-back'}).append(theImageControl.image);
+            let textDiv = $("<div>").append(theTextControl.textArea);
+            if(mentorMode) {
+                let topTextDiv = textDiv;
+                let bottomTextDiv = $("<div>", {class: 'image-back'});
+                textDiv = $("<div>").append(topTextDiv, bottomTextDiv);
             }
+            stretchDiv.append(imageDiv, textDiv);
+            let theSplitter = viewSplitter(stretchDiv);
+            if(mentorMode) {
+                const subSplitID = "sub_split_percent";
+                let subSplitPercent = localStorage.getItem(subSplitID);
```

Comment:
This was already present, but since local storage only stores strings, here subSplitPercent can be a string or a number (line 123). While javascript allows this, it isn't ideal for JIT (not big deal here) or consistency. I'd use a `parseFloat` on the response from `localStorage.getItem`

Reply:
Good call -- I think that's clearer.